lib/index.js

```diff
@@ -168,14 +183,14 @@ class WebpackPluginServe extends EventEmitter {
     });
 
     watchRun.tapPromise(key, async () => {
-      if (!this.startPromise) {
+      if (!this.state.starting) {
         // ensure we're only trying to start the server once
-        this.startPromise = start.bind(this)();
-        this.startPromise.then(() => newline());
+        this.state.starting = start.bind(this)();
+        this.state.starting.then(() => newline());
       }
 
       // wait for the server to startup so we can get our client connection info from it
-      await this.startPromise;
+      await this.state.starting;
```

Comment:
This wouldn't already be resolved on line 189?

Reply:
Nope